python/erdos/__init__.py

```diff
@@ -127,9 +127,9 @@ def connect_sink(
     if isinstance(read_stream, Stream):
         py_stream = read_stream._py_stream
     elif isinstance(read_stream, IngestStream):
-        py_stream = read_stream._py_ingest_stream.to_py_stream()
+        py_stream = read_stream._py_ingest_stream
     elif isinstance(read_stream, LoopStream):
-        py_stream = read_stream._py_loop_stream.to_py_stream()
+        py_stream = read_stream._py_loop_stream
```

Comment:
Can we consolidate them now? We don't need to convert the streams anymore so we can just save the internal Python stream classes in all kinds of stream in `read_stream._py_stream` and skip this check.